test/BondFactory.spec.ts

```diff
@@ -31,28 +35,50 @@ describe("BondFactory", async () => {
     ({ collateralToken, paymentToken } = await (
       await tokenFixture([18])
     ).tokens[0]);
+
     ISSUER_ROLE = await factory.ISSUER_ROLE();
   });
 
-  async function createBond(factory: BondFactory) {
+  async function createBond(factory: BondFactory, params: any = {}) {
+    const {
```

Comment:
Good idea making this with defaults - I would rather these not be destructured as they're not used again and can be used as such
`const testCollateralToken = params.collateralToken || collateralToken.address`